administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -83,8 +85,10 @@
 				</tr>
 			</tfoot>
 			<tbody>
-			<?php foreach ($this->items as $i => $item) : ?>
-				<?php // @todo ACL check ?>
+			<?php foreach ($this->items as $i => $item) :
+				$canEdit    = $user->authorise('core.edit', $this->state->get('component') . $item->id);
+				$canEditOwn = $user->authorise('core.edit.own', $this->state->get('component') . $item->id) && $item->created_by == $userId;
```

Comment:
- Does this work for a component that does not have this ACL (`core.edit`, 'core.edit.own') permissions?
- Does this work for a component that does not have item level permissions (ex: com_menus), i.e., just permissions?
- Isn't needed to have more permissions checked? Other components check more things.